apps/io-wallet-user-func/src/infra/http/handlers/create-wallet-instance.ts

```diff
@@ -94,7 +88,7 @@ export const CreateWalletInstanceHandler = H.of((req: H.HttpRequest) =>
                 "NEW_WALLET_INSTANCE_CREATED",
               ),
             ),
-            RTE.chainW(() => sendEmail(walletInstance.userId)),
+            RTE.chainW(() => sendCreationEmail(walletInstance.userId)),
```

Comment:
Maybe we can do
```suggestion
            RTE.chainW(() => enqueue(walletInstance.userId)),
```
and remove `sendCreationEmail` function